receiver/pulsarreceiver/factory.go

```diff
@@ -26,33 +26,6 @@ const (
 // FactoryOption applies changes to PulsarExporterFactory.
 type FactoryOption func(factory *pulsarReceiverFactory)
```

Comment:
@atoulme, looks like the `checkapi` tool missed `FactoryOption`, `TracesUnmarshaler`, `MetricsUnmarshaler` and `LogsUnmarshaler`. Is this expected?

Reply:
yes it is. For now, the checkapi tool only checks functions, not types. We can add that next but it's more complicated, as Config structs must be exported.